.github/workflows/ci_linux_x64_clang.yml

```diff
@@ -42,9 +45,19 @@ jobs:
           submodules: true
       - name: Install Python requirements
         run: python3 -m pip install -r ./runtime/bindings/python/iree/runtime/build_requirements.txt
+      - name: Setting up Cache
+        run: |
+          mkdir -p /mnt/azureblob
+          python3 ./build_tools/ccache/edit_fuse_connection.py ${{ secrets.AZURE_CCACHE_CONTAINER_KEY }}
+          blobfuse2 mount --allow-other --config-file=./build_tools/ccache/fuse_connection2.yaml /mnt/azureblob/
+          export SCCACHE_DIR=/mnt/azureblob/ccache-container
```

Comment:
Shouldn't need this mount or any of the `build_tools/ccache/` files when using sccache, unless we use the mount for anonymous read access. I'd start with the authenticated read/write access.